src/components/Map/PlaceSection/index.tsx

```diff
@@ -1,27 +1,44 @@
-import { useMemo } from 'react';
+import { useEffect, useMemo, useRef } from 'react';
 import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 import PlaceItem from '@/components/Map/PlaceSection/PlaceItem';
-import { PlaceInfo, LocationData } from '@/types';
+import { PlaceData, LocationData } from '@/types';
 import { useGetPlaceList } from '@/api/hooks/useGetPlaceList';
+import Loading from '@/components/common/layouts/Loading';
 
 interface PlaceSectionProps {
   mapBounds: LocationData;
   filters: {
     categories: string[];
     influencers: string[];
-    location: { main: string; sub?: string };
+    location: { main: string; sub?: string; lat?: number; lng?: number };
   };
-  onPlacesUpdate: (places: PlaceInfo[]) => void;
+  onPlacesUpdate: (places: PlaceData[]) => void;
+  center: { lat: number; lng: number };
+  shouldFetchPlaces: boolean;
+  onFetchComplete: () => void;
+  initialLocation: boolean;
 }
 
-export default function PlaceSection({ mapBounds, filters, onPlacesUpdate }: PlaceSectionProps) {
+export default function PlaceSection({
+  mapBounds,
+  filters,
+  onPlacesUpdate,
+  center,
+  shouldFetchPlaces,
+  onFetchComplete,
+  initialLocation,
+}: PlaceSectionProps) {
   const navigate = useNavigate();
-  const { data: placeList } = useGetPlaceList(mapBounds, filters);
+  const previousPlacesRef = useRef<PlaceData[]>([]);
+
+  const { data: places, isLoading, isError, error, refetch } = useGetPlaceList(mapBounds, filters, center, false);
```

Comment:
```suggestion
  const { data: places, isLoading, isError, error, refetch } = useGetPlaceList(mapBounds, filters, center, true);
```

enabled를 false로 고정시켜두면 useQuery 호출이 비활성화 된 채로 동작하지 않을거라 이 부분은 추후 까먹지 않고 개선 부탁드릴게요.